tests/test_core.py

```diff
@@ -7903,6 +7903,72 @@ def test_asyncify_indirect_lists(self, args, should_pass):
       if should_pass:
         raise
 
+  @needs_dylink
+  @no_memory64('TODO: asyncify for wasm64')
+  def test_asyncify_side_module(self):
+    self.set_setting('ASYNCIFY')
+    self.set_setting('EXIT_RUNTIME', 1)
+    self.emcc_args += ['-sASYNCIFY_IMPORTS=["_Z8my_sleepi"]']
```

Comment:
The file could be C which would avoid name mangling here (pass `force_c=True` to `dylink_test`)

Reply:
Sorry, I think a typo swallowed parts of my previous comments. The function being added isn't `setWasmTableEntry` - `setWasmTableEntry` is where the adding fails because no signature can be found. I clarified the above comment with bold/italic changes.

Attached the debugger at this point in question. You'll see that the function doesn't have a signature and is already added to GOT.
![image](https://user-images.githubusercontent.com/748698/164894488-9d076f45-6e37-474a-8e45-53fbedd90e05.png)